en/docs/quick-start-guides/deploy-your-first-automation.md

```diff
@@ -0,0 +1,119 @@
+# Deploy Your First Automation
+
+## Overview
+
+In this guide, you will:
+
+- Create a Devant project and add a simple automation which will print `"Hello World"` every minute .
```

Comment:
Let's not create a new project, let's make it simple as possible. Use default project instead